workspace/testing/tests/test_appeals_has.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import pipelineutils
+import constants
+import warnings
+
+def test_appeal_event_notebook(credential_name, azure_credential, synapse_endpoint: str):
```

Comment:
The function needs to be named correctly.

Reply:
pushed a commit to fix this